scripts/cleanup_e2e_resources.ts

```diff
@@ -424,3 +434,52 @@ for (const staleRole of allStaleRoles) {
     );
   }
 }
+
+const listAllStaleSSMParameters = async (): Promise<
+  Array<ParameterMetadata>
+> => {
+  let nextToken: string | undefined = undefined;
+  const parameters: Array<ParameterMetadata> = [];
+  do {
+    const describeParametersCommandOutput: DescribeParametersCommandOutput =
+      await ssmClient.send(
+        new DescribeParametersCommand({
+          NextToken: nextToken,
+          MaxResults: 50,
+          ParameterFilters: [
+            {
+              Key: 'Name',
+              Option: 'BeginsWith',
+              Values: ['/amplify/'],
+            },
+          ],
+        })
+      );
+    nextToken = describeParametersCommandOutput.NextToken;
+    if (describeParametersCommandOutput.Parameters) {
+      describeParametersCommandOutput.Parameters.filter(
+        (parameter: ParameterMetadata) => isStale(parameter.LastModifiedDate)
+      ).forEach((parameter: ParameterMetadata) => {
+        parameters.push(parameter);
+      });
+    }
+  } while (nextToken);
+  return parameters;
+};
+
+const allStaleSSMParameters = await listAllStaleSSMParameters();
+for (const staleSSMParameter of allStaleSSMParameters) {
+  try {
+    await ssmClient.send(
+      new DeleteParameterCommand({
+        Name: staleSSMParameter.Name,
+      })
+    );
```

Comment:
I was debating between `DeleteParameterCommand` and `DeleteParametersCommand` 
but the latter would need some chunking anyway as request size is not infinite. 
So deleting one by one for simplicity.
Perf shouldn't matter if script runs every hour and deletes params accumulated in last 3 hours anyway.